app/jobs/sms_sender_confirmation_job.rb

```diff
@@ -4,7 +4,7 @@ class SmsSenderConfirmationJob < ActiveJob::Base
   def perform(code, mobile)
     TwilioService.new.send_sms(
       to: mobile,
-      body: "Your #{APP_NAME} phone confirmation code is: #{code}."
+      body: "#{code} is your #{APP_NAME} one-time passcode."
```

Comment:
I don't think we need different messages -- from the user standpoint, they just want to read the number and type in the computer; yes, context is diff but i don't think that needs to translate to a diff message